packages/@aws-cdk/aws-cloudwatch/README.md

```diff
@@ -302,6 +302,17 @@ dashboard.addWidgets(new GraphWidget({
 }));
 ```
 
+The graph can show the value from the entire time range.
+
+```ts
+dashboard.addWidgets(new GraphWidget({
+  // ...
+  // ...
+
+  setPeriodToTimeRange: true,
+}));
+```
+
```

Comment:
The documentation on the property is sufficient, no need to add it in the README.  I have removed the lint requirement so the check will pass,

Reply:
Hi, I really need this field, any plan to merge? 
I see this is idle since many days, If needed I'm available to work on this

Reply:
```suggestion
```